war/src/main/js/widgets/config/tabbar.less

```diff
@@ -109,7 +109,6 @@
     border: 1px solid var(--input-border);
     padding: 2rem 2rem 0 2rem;
     border-top: none;
-    border-bottom: none;
```

Comment:
This part solely affects the buttons if they are at the end of the page:

https://user-images.githubusercontent.com/13383509/158563402-3e551ceb-b938-4992-8846-1bd9939ccf66.mp4

However, the buttons are sticky and move if you scroll up or down, hence the change proposed doesn't address the issue fully.
On previous versions, the buttons were honored in a separate box at the bottom left of the configuration tab:
![](https://i.imgur.com/Ewv0J9p.png)
Maybe we can reintroduce it looking more modern, thoughts?

Reply:
The bar sticks to the bottom of your screen and moves up and down depending on your scroll behavior.
The current change proposed solely takes care of the white bar at the bottom of the screen, it doesn't address the scrolling part.
I admit, the wording I've chosen was a bit misleading, apologies.

Reply:
The bar sticks to the bottom of your screen and moves up and down depending on your scrolling behavior.
The current change proposed solely takes care of the bar if you are at the end of the configuration box, it's still visible at the scrolling part.
I admit, the wording chosen was a bit misleading, apologies.

Reply:
The upper border line cannot be seen when you scrolling at configuration box. If border-bottom is added when it is not scrolling to the bottom, it will be inconsistent up and down.

Reply:
Isn't it the scrollspy tabs that provide the top border?

Reply:
![動畫](https://user-images.githubusercontent.com/97017043/158989852-dbe98eb1-c02e-4d2c-9499-50c591f60abb.gif)
@daniel-beck The top border is invisible when you are not at the top.